src/main/java/io/swagger/oas/inflector/controllers/OpenAPIOperationController.java

```diff
@@ -296,182 +296,183 @@ public Response apply(ContainerRequestContext ctx) {
-                                                            for (CSVRecord innerKVPair : innerString) {
-                                                                if (innerKVPair.size() == 2) {
-                                                                    String key = innerKVPair.get(0).trim();
-                                                                    String value = innerKVPair.get(1).trim();
-                                                                    if ("name".equals(key)) {
-                                                                        name = value;
+                if (body.getContent() != null) {
+                    for (String mediaType : body.getContent().keySet()) {
+                        if (formDataString == null) {
+                            // can only read stream once
+                            if (mt.isCompatible(MediaType.MULTIPART_FORM_DATA_TYPE)) {
+                                // get the boundary
+                                String boundary = mt.getParameters().get("boundary");
+
+                                if (boundary != null) {
+                                    try {
+                                        InputStream output = ctx.getEntityStream();
+
+                                        MultipartStream multipartStream = new MultipartStream(output, boundary.getBytes());
+                                        boolean nextPart = multipartStream.skipPreamble();
+                                        while (nextPart) {
+                                            String header = multipartStream.readHeaders();
+                                            // process headers
+                                            if (header != null) {
+                                                CSVFormat format = CSVFormat.DEFAULT
+                                                        .withDelimiter(';')
+                                                        .withRecordSeparator("=");
+
+                                                Iterable<CSVRecord> records = format.parse(new StringReader(header));
+                                                for (CSVRecord r : records) {
+                                                    for (int j = 0; j < r.size(); j++) {
+                                                        String string = r.get(j);
+
+                                                        Iterable<CSVRecord> outerString = CSVFormat.DEFAULT
+                                                                .withDelimiter('=')
+                                                                .parse(new StringReader(string));
+                                                        for (CSVRecord outerKvPair : outerString) {
+                                                            if (outerKvPair.size() == 2) {
+                                                                String key = outerKvPair.get(0).trim();
```

Comment:
code from 328 - 338 and from 340 - 350 are basically the same with a small different, can you create a method to wrap that instead having this repeated?